extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -163,6 +167,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				await this.showPreviewFile(urlToOpen);
 			}
 
+			TelemetryReporter.createActionEvent(NotebookTelemetryView, NbTelemetryActions.OpenBook).send();
```

Comment:
During our discussion you mentioned that you were intending on seeing data about how users were opening books - but this is going to get called from a variety of places. You should consider either putting the telemetry call higher up and/or being more specific about how the book was opened